### PR TITLE
Enable returning support size with exact algorithms for alpha=1.5 and alpha=2

### PR DESCRIPTION
**Pull request by bpopeters**
This pull request enables the support size (the number of nonzero elements in the probability vector predicted by entmax) to be returned as a second output from `entmax.entmax15`, `entmax.sparsemax`, and their respective loss functions. This is fast because the support sizes are already computed inside the exact algorithm, it's just a question of exposing them to the user. This pull request does NOT implement returning support for `entmax_bisect` because the bisection-based algorithm does not explicitly compute the support size, so it cannot be returned for free.

The main application of this feature is for logging and benchmarking. The accompanying benchmarking scripts show how strongly the runtime of the topk algorithm depends on the support size.

```
In [1]: z = torch.tensor([5, 5, 0])

In [2]: entmax15(z)
Out[2]: tensor([0.5000, 0.5000, 0.0000])

In [3]: entmax15(z, return_support=True)
Out[3]: (tensor([0.5000, 0.5000, 0.0000]), tensor([2]))

```

### CODE REVIEW

**Comment by vene** [comment]
I would call this "return_support_size", because in a future update we may
want to return the actual support.

**Comment by bpopeters** [comment]
@vene In order to make tests pass, I needed to increase the minimum version to 1.9 (previously it was 1.3). Do you think this is reasonable (1.9 is still quite ancient), or should I try to figure out where precisely in the code I angered the gods of backwards compatibility?

**Comment by vene** [comment]
I think this is reasonable. The previous version of this package would
remain usable for anyone constrained to use older pytorch, and we don't
have the capacity to preempt this hopefully rare case.
